lib/xmlparser.rb

```diff
@@ -123,6 +125,11 @@ def initialize
     # real id updates
     @room_id = nil
     @previous_nav_rm = nil
+
+    # claim update
+    @arrival_pcs = []
```

Comment:
This seems like something we should make an attr_reader so it's able to be read from outside this module probably?

Reply:
attr_accessored arrival_pcs and the obvious_hidden_player values